tests/upgrades/test_performance_tuning.py

```diff
@@ -122,14 +179,54 @@ def _create_custom_hiera_file(mongodb_type, tune_size="default"):
             if tune_size == "medium":
                 fd.write("\n".join(MEDIUM_TUNING_DATA))
 
+    @staticmethod
+    def _data_collection_of_set_tune_params(
+        tune_size_data, tune_params_collection_regex, tune_params_group
+    ):
+        """
+        This method is used to create the set of expected tune parameters value from the
```

Comment:
I think this can be transformed into :
`This method creates as well as reads the tune parameters on satellite server.`